core-services/jira-lifecycle-plugin/config.yaml

```diff
@@ -16,6 +16,8 @@ default:
     - resolution: DONE-ERRATA
       status: CLOSED
     is_open: true
+    require_release_notes: true
+    release_notes_default_text: "Cause: What actions or circumstances cause this bug to present.\r\nConsequence: What happens when the bug presents.\r\nFix: What was done to fix the bug.\r\nResult: Bug doesn’t present anymore."
```

Comment:
Do we need to announce this somewhere?

Reply:
The announcement email has been sent and this requirement is ready to be enabled.